packages/adapter-node/index.js

```diff
@@ -78,6 +79,28 @@ export default function ({
 			const buildOptions = esbuildConfig ? await esbuildConfig(defaultOptions) : defaultOptions;
 			await esbuild.build(buildOptions);
 
+			utils.log.minor('Building SvelteKit servers reference server');
```

Comment:
don't need server in the message twice
```suggestion
			utils.log.minor('Building SvelteKit reference server');
```

Reply:
fixed